client/src/components/inventory/inventory.js

```diff
@@ -13,14 +13,22 @@ class Inventory extends Component {
     })};
 
     render() {
-        return this.state.map(item =>(
+        return this.state.inventory.map(item =>(
             <div>
-                <h2>Inventory</h2>
-                <input type="text" placeholder="Search"/>
-                <p>{item}</p>
+                <p>ITEM</p>
```

Comment:
shouldnt be all p tags, need classnames

Reply:
Yeah I know I'm just getting the data on the page